src/mlagility/cli/cli.py

```diff
@@ -434,6 +442,37 @@ def main():
             sys.argv.insert(1, "benchmark")
 
     args = parser.parse_args()
+    if args.func == benchmark_command:
+        # Validate runtime arg
+        if args.runtimes:
+            if len(args.runtimes) != len(args.devices):
+                raise argparse.ArgumentError(
+                    argparse_runtimes, "The number of devices and runtimes must match"
```

Comment:
This would break the single device and multiple runtime case

Reply:
I chatted with @jeremyfowers and we agreed that supporting both multiple devices and multiple runtimes on a single CLI call was confusing. We are now only allowing for a single device + one or more runtimes.

Therefore, this exception has been removed